File: Android/JavaPic/app/src/main/java/org/uestc1211/myapplication/MainActivity.java
package org.uestc1211.myapplication;

import androidx.appcompat.app.AppCompatActivity;

import android.graphics.Bitmap;
import android.graphics.BitmapFactory;
import android.graphics.drawable.Drawable;
import android.os.Bundle;
import android.util.TypedValue;
import android.widget.Button;
import android.widget.ImageView;
import android.widget.TextView;

import java.util.Timer;
import java.util.TimerTask;

public class MainActivity extends AppCompatActivity {

    private TextView textView;
    private ImageView imageView;
    private Button button;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        textView= findViewById(R.id.textView);
        imageView= findViewById(R.id.imageView);

        imageView.setImageResource(R.drawable.a00);

        Bitmap bmp = BitmapFactory.decodeResource(getResources(),R.drawable.a02);
        imageView.setImageBitmap(bmp);
        Drawable drawable = getResources().getDrawable(R.drawable.a03);

        button=findViewById(R.id.button);
        button.setText("年末的歌吹啊工资");
        button.setTextColor(0xff00ff00);
        button.setTextSize(TypedValue.COMPLEX_UNIT_SP,26);



    }
}

File: Android/decades/app/src/main/java/org/uestc1123/decades/MainActivity.kt
package org.uestc1123.decades

import android.app.Activity
import android.os.Bundle
import android.view.View
import android.widget.ImageView
import android.widget.RelativeLayout
import androidx.appcompat.app.AppCompatActivity

class MainActivity: Activity(){
    private var images= intArrayOf(R.drawable.ic_launcher_background,R.drawable.ic_launcher_foreground)
    private var currentImage = 0
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val main = findViewById<RelativeLayout>(R.id.root)
        val image=ImageView(this)
        main.addView(image)
        image.setImageResource(images[0])
        image.setOnClickListener{
            image.setImageResource(images[++currentImage%images.size])
        }
    }
}
